### DISCUSSION TITLE
"Error: Requested chains are not supported" when trying to connect with WalletConnect V2

### DISCUSSION THREAD

**Discussion by GabrielBuragev** [trustwallet/wallet-core#3277] [Q&A]
**Describe the bug**
When trying to connect with WalletConnect V2 (after scanning the QR code) the following error is thrown:
`exception Error: Requested chains are not supported`

I tested the same implementation on MetaMask app `v7.0.1 (1126)` and it works.

**To Reproduce**
Using Android 11 and TrustWallet v7.25 (from Google Play Store) i was able to reproduce this.
Created a minimal reproducible environment here: https://codesandbox.io/s/sleepy-silence-7nvgjj?file=/src/services/connectToWalletConnect.js

Just try connecting to this app via TrustWallet.

**Expected behavior**
I expect that a connection is successfully established after scanning the QR code.

**Screenshots**
![Screenshot 2023-06-27 at 15 52 09](https://github.com/trustwallet/wallet-core/assets/18466333/1253eece-1f3a-4c1d-a2c5-983df6e2c594)

**Additional context**

**Comment by zhengyimeng** [comment]
如何解决这个问题？

**Comment by harshmittal1750** [comment]
Any solution for this?
